YAXLib/Custom/IMemberContext.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (C) Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors.
```

Comment:
This is a member of `ISerializationContext`, and provides all member-specific information.
Before, `Type` and `Member`-specific information were included side-by-side.